docs/building.md

```diff
@@ -11,7 +11,24 @@
   ninja 1.10.0
   Xilinx Vitis 2021.2 
 ```
+### Regular packages
+```
+sudo apt install ninja-build clang-10 clang lld-10 lld
+```
+### cmake
+Ubuntu 20.04 is released with:
+cmake/focal,now 3.16.3-1ubuntu1 amd64 [installed]
+
+But we require:
+3.20.6 (or newer)
+```
+wget https://cmake.org/files/v3.20/cmake-3.20.6-linux-x86_64.sh
+chmod +x cmake-3.20.6-linux-x86_64.sh
+sudo ./cmake-3.20.6-linux-x86_64.sh --prefix=/usr/local
+cmake --version
```

Comment:
I usually just pip install cmake with any other requirements. It's easier than building it and doesn't require any special permissions or `PATH`.

Reply:
That's fine - would you like to update the instructions with your method?